package.json

```diff
@@ -101,5 +101,8 @@
   ],
   "kaltura": {
     "name": "uiManagers"
+  },
+  "dependencies": {
+    "@playkit-js/common": "^1.1.1"
```

Comment:
did you make sure that only `<A11ly/>` was imported and added to the bundle  using [tree-shaking](https://webpack.js.org/guides/tree-shaking/) ?

Reply:
done
related PR for ui-common repo: https://github.com/kaltura/playkit-js-common/pull/10